packages/apollo-link-schema/README.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Schema Link
+---
+
+## Purpose
+An Apollo Link to allow mocking and server rendering
+
+## Installation
+
+`npm install apollo-link-schema --save`
+
+
+## Usage

```

Comment:
We should likely add an SSR example. I'm not 100% sure off hand what that would look like.

Reply:
It solves/codifies this problem.

https://www.apollographql.com/docs/react/recipes/server-side-rendering.html#local-queries

Should we just update those docs to use `SchemaLink` and point it's usage there?